service/__init__.py

```diff
@@ -0,0 +1,32 @@
+"""
```

Comment:
isnt this file also in your other PR? we dont want to have a merge conflict

Reply:
You're right. I kept working on this story after I finished the last one and didn't start from where the origin/master was